test/end-to-end/mvFeatures/redact.js

```diff
@@ -47,6 +49,18 @@ Scenario('Redact text and then removing the redaction', async ({I}) => {
 }).tag('@ci')
   .retry(testConfig.TestRetryScenarios);
 
+Scenario('Redact first page', async ({I}) => {
+  await redactFirstPageTest(I, mvData.REDACTION_CASE, mvData.PDF_DOCUMENT);
+
+}).tag('@ci')
+  .retry(testConfig.TestRetryScenarios);
+
+Scenario('Redact multiple pages', async ({I}) => {
+  await redactMultiplePagesTest(I, mvData.REDACTION_CASE, mvData.PDF_DOCUMENT);
+
+}).tag('@ci')
```

Comment:
update tag name as "@np" (nightly pipeline)  .

This test won't run until the code is deployed on AAT  & 
moreover AAT MV execution using  different test data so you need to update the code according AAT & Preview test data.

For further details look at AAT Test Data case:
![image](https://user-images.githubusercontent.com/87771652/212659182-a3a7ceeb-40eb-483d-91dc-00bb673e0cd9.png)

Reply:
I had a call with Michael and the new feature tests will run both AAT as well as on Preview